content/event-platform/available-events.md

```diff
@@ -1676,6 +1676,605 @@ The modifications that can lead to this event include:
+| data.product_model.changes.associations.removed.products (single item) | string | - |
+| data.product_model.changes.associations.removed.productModels | array&lt;string&gt; | - |
+| data.product_model.changes.associations.removed.productModels (single item) | string | - |
+| data.product_model.changes.quantified_associations | object | - |
+| data.product_model.changes.quantified_associations (additional properties) | object | - |
+| data.product_model.changes.quantified_associations.added | object | - |
+| data.product_model.changes.quantified_associations.added.products | array&lt;object&gt; | - |
+| data.product_model.changes.quantified_associations.added.products.uuid | string | - |
+| data.product_model.changes.quantified_associations.added.products.quantity | number | - |
+| data.product_model.changes.quantified_associations.added.productModels | array&lt;object&gt; | - |
+| data.product_model.changes.quantified_associations.added.productModels.code | string | - |
+| data.product_model.changes.quantified_associations.added.productModels.quantity | number | - |
+| data.product_model.changes.quantified_associations.removed | object | - |
+| data.product_model.changes.quantified_associations.removed.products | array&lt;string&gt; | - |
+| data.product_model.changes.quantified_associations.removed.products (single item) | string | - |
+| data.product_model.changes.quantified_associations.removed.productModels | array&lt;string&gt; | - |
+| data.product_model.changes.quantified_associations.removed.productModels (single item) | string | - |
+| data.product_model.changes.values | object | - |
+| data.product_model.changes.values (additional properties) | array&lt;object&gt; | - |
+| data.product_model.changes.values.previous | string \| number \|
+| data.product_model.changes.values.new | string \| number \|
+| data.product_model.changes.values.type | string | - |
+| data.product_model.changes.values.locale | string \| null |
+| data.product_model.changes.values.scope | string \| null |
+| data.product_model.changes.table_attributes | object | Changes to table attributes. |
+| data.product_model.changes.table_attributes.added | array&lt;object&gt; | - |
+| data.product_model.changes.table_attributes.added (single item) | object | - |
+| data.product_model.changes.table_attributes.removed | array&lt;object&gt; | - |
+| data.product_model.changes.table_attributes.removed (single item) | object | - |
+| data.author | object | - |
+| data.author.identifier | string | Identifier of the author. |
+| data.author.type | string | Type of the author. |
+
+<details>
+  <summary style="cursor: pointer;">Click here to expand the payload example</summary>
+> Example of payload
+
+
+```json [snippet:Payload]
+
+{
+  "specversion": "1.0",
+  "id": "018e32f9-dfe4-760e-a273-5da1c089dfdb",
+  "type": "com.akeneo.pim.v1.product-model.updated.delta",
+  "source": "pim",
+  "subject": "srnt-whatever",
```

Comment:
```suggestion
  "subject": "019256aa-7f77-7f58-bc8c-770f7e369c5b",
```

out of the event platform, the subject is the subscription uuid, not the pim tenant id